lib/core/rollup.config.js

```diff
@@ -0,0 +1,280 @@
+import fs from "fs";
+import path from "path";
+import vue from "rollup-plugin-vue";
+import alias from "@rollup/plugin-alias";
+import commonjs from "@rollup/plugin-commonjs";
+import resolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import babel from "@rollup/plugin-babel";
```

Comment:
hm are you sure we need babel? I mean using the "lib" and "target" options in TSconfig can help us pretty much